daos/mysql/Items.php

```diff
@@ -478,4 +478,32 @@ public function stats() {
             FROM '.\F3::get('db_prefix').'items;');
         return $res[0];
     }
+
+
+    /**
+     * returns the datetime of the last item update or user action in db
+     *
+     * @return timestamp
+     */
+    public function lastUpdate() {
+        $res = \F3::get('db')->exec('SELECT
+            MAX(updatetime) AS last_update_time
+            FROM '.\F3::get('db_prefix').'items;');
+        return $res[0]['last_update_time'];
+    }
+
+
+    /**
+     * returns the statuses of items last update
+     *
+     * @param date since to return item statuses
+     * @return array of unread, starred, etc. status of specified items
+     */
+    public function statuses($since) {
+        $res = \F3::get('db')->exec('SELECT id, unread, starred
```

Comment:
Is it possible to use proper booleans? Otherwise the sync will mark all changed items as starred and unread since unlike in PHP `!!"0" === true` in JS:

```shell
$ php -r 'var_dump(!!"0");'
Command line code:1:
bool(false)

$ node -e 'console.log(!!"0")'
true
```